README.md

```diff
@@ -78,6 +78,24 @@ Get json by taking the name and returning the dictionary
 r.get_json("json")
 ```
 
+### Pandas DataFrame supported
+Note that json is used to serialize which comes with other caveats that can be found here: https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.DataFrame.to_json.html
```

Comment:
I think this makes sense to include in the docstring for the function itself too.

Info about doc strings:

https://www.python.org/dev/peps/pep-0257/

We can follow up on another PR though.